umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -46,22 +52,31 @@ If you send the messages into other topics, some features might not work correct
 {{% /notice %}}
 
 ## What are the limitations?
+- Increased complexity in integrating MQTT with Kafka.
+- Deviating from established standards might initially cause discomfort among some users.
```

Comment:
Why do we deviate from established standards?

Reply:
Because we use a specific data model (UMH datamodel). It is mentioned in [this page](https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/#Topic_Hierarchiy), but I remove it because the sentence below explains the topic structure.